src/Run/Setup.cs

```diff
@@ -149,6 +149,12 @@ private CompleteCommand BuildCommand(string commandSelectedByUser, List<string>
         private string BuildParametersForCommand(Dictionary<string, string> commandParameters, string toolName)
         {
             string commandSetting = string.Empty;
+
+            if (Tools.ContainsKey(toolName))
```

Comment:
I would also check whether or not this "defaultParameters" is null or empty to allow for people to omit it from the config file.